tests/prom/alerts-modified.yaml

```diff
@@ -0,0 +1,68 @@
+
+groups:
+  - name: dis-service
+    rules:
+      - alert: NoDataReceived
+        expr: sum(rate(processor_sent{namespace="dev", container="dis", label="convert_cloudevent"}[30m])) BY (pod, job, namespace) == 0
```

Comment:
probably should not be there under benthos folder

Reply:
it's under the prom folder

Reply:
but I want to ignore this file

Reply:
Probably you do commits so quick)